demos/notebooks/demo_pipeline_cnmfE.ipynb

```diff
@@ -264,22 +372,69 @@
     "                                'min_corr': min_corr,\n",
     "                                'min_pnr': min_pnr,\n",
     "                                'normalize_init': False,               # just leave as is\n",
-    "                                'center_psf': True,                    # leave as is for 1 photon\n",
+    "                                'center_psf': True,                    # True for 1p\n",
     "                                'ssub_B': ssub_B,\n",
     "                                'ring_size_factor': ring_size_factor,\n",
     "                                'del_duplicates': True,                # whether to remove duplicates from initialization\n",
     "                                'border_pix': bord_px})                # number of pixels to not consider in the borders)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cnmfe_model = cnmf.CNMF(n_processes=n_processes, \n",
+    "                        dview=cluster, \n",
+    "                        params=parameters)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "    <h2 >CNMF-E: The Ring Model</h2>\n",
+    "   Background activity is very ill-behaved with 1p recordings: it often fluctuates locally and is much larger in magnitude than the neural signals we want to extract. In other words, the large-scale background model used for CNMF is not sufficient for most 1p data. Hence, Pengcheng Zhou and others came up with a localized model of background activity for CNMFE: CNMFE represents the background at each pixel as the weighted sum of activity from a circle (or ring) of pixels a certain distance from that pixel. The distance of this ring from the reference pixel is set by the <em>ring_size_factor</em> parameter. This more complex pixel-wise background model explains why CNMFE is computationally more expensive than CNMF, and also why it works better to mop up large-scale localized background noise to find the neurons in your 1p data. \n",
```

Comment:
"very" -> ""
"often" -> ""
"much" -> ""
"In other words, " -> ""
"Hence, " -> ""
"came up with" -> "defined" (or designed, or some other word)
"why it works better to mop up" -> "why it is more effective at removing"